packages/backend/test/utils.ts

```diff
@@ -426,10 +478,18 @@ export const vote = async (
 }
 
 export const epochTransition = async (t) => {
-    const { txManager, unirep } = t.context
-    const calldata = unirep.interface.encodeFunctionData('beginEpochTransition')
-    const hash = await txManager.queueTransaction(unirep.address, calldata)
-    await t.context.unirep.provider.waitForTransaction(hash)
+    const prevEpoch = await t.context.unirep.currentEpoch()
+    const { EpochManager } = require('../src/daemons/EpochManager')
```

Comment:
I'd prefer not to use the `EpochManager` here. We have separate tests for this daemon. This function should interact directly with the blockchain to prevent odd behaviour.